### ISSUE TITLE
ArgumentNullException

### ISSUE THREAD

**Issue by yongin-jo** [OYIon/LiveSharp#48]
I just created xamarin.forms project and installed LiveSharp from nuget.
Visual Studio 2019 and Rider show me error mesage below.

```  
LiveSharp.targets(26, 5): LiveSharp exception: System.ArgumentNullException: Value cannot be null.
Parameter name: method
  at Mono.Cecil.Mixin.CheckMethod (System.Object method) [0x00012] in <02cd44f94ebd4355b3be9e1dbd9d77a1>:0 
  at Mono.Cecil.ModuleDefinition.ImportReference (Mono.Cecil.MethodReference method, Mono.Cecil.IGenericParameterProvider context) [0x00000] in <02cd44f94ebd4355b3be9e1dbd9d77a1>:0 
  at Mono.Cecil.ModuleDefinition.ImportReference (Mono.Cecil.MethodReference method) [0x00000] in <02cd44f94ebd4355b3be9e1dbd9d77a1>:0 
  at LiveSharp.BuildAgent.Building.LiveSharpContext.ImportRuntimeMethodsInto (Mono.Cecil.ModuleDefinition module) [0x00055] in <95f834bb4089474e843e734de86c992f>:0 
  at LiveSharp.BuildAgent.Building.Builder.ProcessEverything (System.Boolean isFullRebuild) [0x00297] in <95f834bb4089474e843e734de86c992f>:0 
```

**Comment by lhughey** [comment]
I'm also seeing this issue and I'm unable to build the solution with the nuget installed. 
Using v1.1.23 nuget and server v.9.13

**Comment by ionoy** [comment]
Thank you for the feedback!

It seems like LiveSharp.Runtime.dll didn't get referenced. Is there something unusual about the solution structure?

**Comment by GiampaoloGabba** [comment]
i've get the same problem with nuget 1.1.23. Downgrading to 1.1.22 fixed the problem :)

**Comment by lhughey** [comment]
> Thank you for the feedback!
> 
> It seems like LiveSharp.Runtime.dll didn't get referenced. Is there anything unusual about the solution structure?

There is nothing odd about the structure that I can see. It is a XF project with a netstandard, iOS and Droid project. I was able to downgrade to and got things running.

**Comment by ionoy** [comment]
The issue is fixed in 1.1.25. Sorry for the inconvenience.
